pkg/component/tasks.go

```diff
@@ -152,8 +152,7 @@ func DefaultStartTask(conf *TaskConfig) {
 				done()
 			}
 		}()
-		invocation := uint(1)
-		for {
+		for invocation := uint(1); true; invocation++ {
```

Comment:
I'm pretty sure you can leave out that `true`.

```suggestion
		for invocation := uint(1);; invocation++ {
```